my.css

```diff
@@ -0,0 +1,277 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header .header {
+  background: black;
+  position: relative;
+}
+
+.logo {
+  position: absolute;
+  width: 98px;
+  height: 37px;
+  left: 52px;
+  top: 36px;
+  color: white;
+  font-family: Montserrat;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 30px;
+  line-height: 37px;
+}
+
+.ru {
+  display: flex;
+  justify-content: space-around;
+  position: absolute;
+  width: 31px;
+  height: 25px;
+  left: 388px;
+  top: 42px;
+  font-family: Montserrat;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+}
+
+.ru__link {
+  margin-left: 25px;
+  width: 11px;
+  height: 5.5px;
+  text-decoration: none;
+  color: #ffffff;
+}
+
+.burgermenu {
+  position: absolute;
+  left: 1163px;
+  top: 0;
+  width: 60px;
+  height: 60px;
+  background-color: #dcdcdc;
+}
+
+.burger {
+  width: 28px;
+  height: 5px;
+  border-radius: 40%;
+  margin: 22px 0 0 15px;
+  background-color: #000000;
+  display: block;
+}
+
+.burger::before {
+  content: "";
+  position: absolute;
+  width: 28px;
+  height: 5px;
+  border-radius: 40%;
+  margin: 10px 0 0 0;
+  background-color: #000000;
+  display: block;
+}
+
+/* конец шапки */
+/* первая секция */
+.collection {
+  display: flex;
+  height: 900px;
+  background: linear-gradient(
+    90deg,
+    rgb(0, 0, 0) 39%,
+    rgba(255, 255, 255, 1) 39%
+  );
+  margin: 0;
+  padding: 0;
+}
+
+.collection__img {
+  position: absolute;
+  width: 550px;
+  height: 550px;
+  left: 200px;
+  top: 200px;
+}
+
+/* заголовок */
+.newcollection__title {
+  position: absolute;
+  width: 390px;
+  height: 164px;
+  left: 789px;
+  top: 230px;
+}
+
+.newcollection__text {
+  position: absolute;
+  width: 390px;
+  height: 127px;
+  left: 789px;
+  top: 463px;
+}
+
+.newcollection__button {
+  position: absolute;
+  left: 790px;
+  top: 630px;
+  width: 180px;
+  height: 40px;
+  border: 1.5px solid #000000;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#000000" to be "#000" (color-hex-length)](https://app.codacy.com/gh/WebHeroSchool/itog/pullRequest?prid=5556249)